hal/src/dmac/channel/mod.rs

```diff
@@ -8,7 +8,7 @@
 //! requires setting a priority level, as well as enabling or disabling
 //! interrupt requests (only for the specific channel being initialized).
 #![cfg_attr(
-    not(any(feature = "samd11", feature = "samd21")),
+    not(any(feature = "samd11", feature = "samd20", feature = "samd21")),
```

Comment:
I would prefer `feature = "min-samd51g"` in this case.

Reply:
Perfect, I didn't know that was equivalent :)
Updated